daemon/logger/copier.go

```diff
@@ -100,6 +113,17 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 					if logErr := c.dst.Log(msg); logErr != nil {
 						logrus.Errorf("Failed to log msg %q for logger %s: %s", msg.Line, c.dst.Name(), logErr)
 					}
+					// If hasMorePartial, this is lastPartial
+					if hasMorePartial {
+						ordinal++
+						setLogMetaData(msg, partialid, ordinal, true)
+
+						// reset
```

Comment:
Are we supposed to reset after adding to the partial?

Reply:
This is the last chunk that's added. The reset is for the local variables used to track `first`, `hasMore` and `ordinal`